tests/csv_upload_tests.py

```diff
@@ -351,29 +269,46 @@ def test_import_into_new_postgres(self):
             if sqlalchemy_utils.database_exists(url):
                 sqlalchemy_utils.drop_database(url)
 
+    def test_create_database_duplicate_name(self):
+        db_name = "examples"
+        error_message = "Error when trying to create Database"
```

Comment:
Couldn't we make this message more specific (or is it already more specific in the logging branch which would lead to this test failing when merging them?)